tools/start-local-vm

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2054  # Arrays are formatted for passing args to other tools
-
 shopt -s nullglob
 
 arch=${BUILDSYS_ARCH}
 variant=${BUILDSYS_VARIANT}
+product_name=bottlerocket
```

Comment:
I suggest initializing with `${BUILDSYS_NAME-bottlerocket}` so in case a buildsys environment variable is already set it takes precedence over the default, similar to how `BUILDSYS_ARCH` and `BUILDSYS_VARIANT` from the environment are considered.

Reply:
Thanks guys for all your comments.

I would rather initialize with `${BUILDSYS_NAME}` because the objective is to allow creating unbranded builds (still compliant with the bottlerocket trademark?) . Moreover it will be consistant with the `Makefile.toml` where their is no constraints on `BUILDSYS_NAME` (no prefix, no suffix, ...).

> > Thanks for the contribution! This is a great addition.
> > I'm thinking `--image-name` or `--image-prefix` might be a bit more accurate in this context. What do you think? We're also happy to make the change if you're busy.
> 
> +1 since this isn't technically a "product" :)

Yes I hesitated also! But in when I saw the comment on `BUILDSYS_NAME` in `Makefile.toml`, I aligned it with the comment ;) : `# Product name used for file and directory naming`.
If I think again that name is to implement the trademark rather a product that an image, no? So I would say stay with that name. We can said that with the script we launch a variant from a product line.

I will push again when all aligned.

Reply:
> I would rather initialize with `${BUILDSYS_NAME}` because the objective is to allow creating unbranded builds (still compliant with the bottlerocket trademark?) . Moreover it will be consistant with the `Makefile.toml` where their are no constraints on `BUILDSYS_NAME` (no prefix, no suffix, ...).

Sorry for not being clearer. This does not add a mandatory "bottlerocket" suffix, but is a form of parameter expansion that provides a default in case a variable is unset (see [this section of POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02)). For example,

```
$ echo ${BUILDSYS_NAME}

$ echo ${BUILDSYS_NAME-bottlerocket}
bottlerocket
$ BUILDSYS_NAME=foobar
$ echo ${BUILDSYS_NAME-bottlerocket}
foobar
```

Hence, if `BUILDSYS_NAME` is already set in the environment, it will transparently be used as `--product-name`/`--image-prefix` (I don't have a strong opinion on either), similar to how that is happening in the various `cargo make` tasks.

Reply:
Oh yes got it, read to fast...my bad. I would use use the more safe ${BUILDSYS_NAME:-bottlerocket} in case it is defined but empty, it could lead to strange name (beginning with a dash)